docs/driver_installation_zadig.md

```diff
@@ -11,6 +11,10 @@ We recommend the use of the [Zadig](https://zadig.akeo.ie/) utility. If you have
 Place your keyboard into the bootloader mode, either by hitting the `RESET` keycode (which may be on a different layer), or by pressing the reset switch usually located on the underside of the board. If your keyboard has neither, try holding Escape, or Space+`B`, as you plug it in (see the [Bootmagic](feature_bootmagic.md) docs for more details).
 Some keyboards may have specific instructions for entering the bootloader, for example the [Bootmagic Lite](feature_bootmagic.md#bootmagic-lite) key (Escape) might be on a different key, such as Left Control. Refer to the board's README if you are unsure.
 
+To put a device in bootloader mode with USBaspLoader you must follow the following sequence: press and hold `BOOT`, press and hold `RESET`, release `RESET`, release `BOOT`.
+An alternative method would be to press and hold `BOOT` while inserting the USB cable.
+To exit bootloader mode press only the `RESET` switch.
```

Comment:
```suggestion
```

Reply:
Up to you on this one, but I put that in there because usbasploader does not automatically exit bootloader mode even after flashing, and reset must be pressed again without the boot jumper. I could mess around with fuse settings to see if I could change that, but plaid and gingham use the default fuses provided by usbasploader, so they'll be this way as well.

Reply:
Most bootloaders don't exit after flashing. The Toolbox for example usually issues a reset command afterwards.